cicd.groovy

```diff
@@ -1,15 +1,22 @@
+// 
```

Comment:
We can probably remove this file now, we'll need to make sure we also update jenkins